packages/components/tree-select/src/select.ts

```diff
@@ -23,6 +23,10 @@ export const useSelect = (
   const result = {
     ...pick(toRefs(props), Object.keys(ElSelect.props)),
     ...attrs,
+    // attrs is not reactive, when v-model binding source changes,
+    // this listener is still old, see the bug(or test 'v-model source change'):
+    // https://github.com/element-plus/element-plus/issues/14204
+    'onUpdate:modelValue': (value) => emit('update:modelValue', value),
```

Comment:
Maybe we can use the constant `UPDATE_MODEL_EVENT`.
https://github.com/element-plus/element-plus/blob/bb12dd5e7eb9ad77c42f6671ed0da59fac04751f/packages/constants/event.ts#L1

Reply:
Sure! Thank you!